File: DailyTask/java/SimpleArithmeticApp.java
import java.util.Random;
import java.util.Scanner;

public class SimpleArithmeticApp{
public static void main(String[] args){
  
Scanner input = new Scanner(System.in);
    
System.out.print("Enter your name: ");
String userName = input.nextLine();
    
Random random = new Random(System.currentTimeMillis());

long start = System.currentTimeMillis();
int score = 0;

for (int index = 0; index < 2; index++){
int number1 = random.nextInt(101);
int number2 = random.nextInt(number1 + 1);
      
int rightAnswer = number1 - number2;
int numberOfTries = 2;

System.out.println("\nWhat is :" + number1 + " - " + number2);

while (numberOfTries> 0) {
 System.out.print("ENter your Answer: ");

try{
        
int answer = Integer.parseInt(input.nextLine());
          
if(answer == rightAnswer){
System.out.println("Correct Anwser");
	score++;
break;
            
}else{

numberOfTries--;
System.out.println("Incorrect anwser you have " + numberOfTries + "left");

}
          
} catch (NumberFormatException e){
          numberOfTries--;
System.out.println("you have " + numberOfTries + "left");

}
}

if (numberOfTries == 0){

System.out.println("Answer was " + rightAnswer);

}
}


System.out.println("\nYour score is : " + score + " / 10");




    
}
}

File: DailyTask/javaScript/simpleArithmeticApp.js


const prompt = require('prompt-sync')();


let userName = prompt('Enter your name: ');

let start = Date.now();
let score = 0;

for (let index = 0; index < 10; index++) {
  let number1 = Math.floor(Math.random() * 101);
  let number2 = Math.floor(Math.random() * (number1 + 1));
  let rightAnswer = number1 - number2;
  let numberOfTries = 2;

  console.log('\nWhat is: ' + number1 + ' - ' + number2);

  while (numberOfTries > 0) {
    let input = prompt('Enter your answer: ');
    try {
      let answer = parseInt(input);
      if (answer === rightAnswer) {
        console.log('Correct answer');
        score++;
        break;
      } else {
        numberOfTries--;
        console.log('Incorrect answer you have ' + numberOfTries + ' left');
      }
    } catch (e) {
      numberOfTries--;
      console.log('you have ' + numberOfTries + ' left');
    }
  }

  if (numberOfTries === 0) {
    console.log('Answer was ' + rightAnswer);
  }
}


console.log('\nYour score is: ' + score + ' / 10');
